Cayenne to DatilDev Name Changes, New Starter Guides (#25)
LIT-Protocol/developer-guides-code: starter-guides/nodejs/README.md

# Getting Started with a Lit SDK Node.js Implementation

> This is a very simple example repository for getting started with a Node.js implementation of the Lit SDK. All it will do is connect an instance of the `LitNodeClient` to the Lit Network. If you are searching for more developed uses of Lit, the [Developer Guides Code](https://github.com/LIT-Protocol/developer-guides-code) repository is for you. 

Reviewer: ```suggestion
This is a very simple example repository for getting started with a Node.js implementation of the Lit SDK. All it will do is connect an instance of the `LitNodeClient` to the Lit Network. If you are searching for more developed uses of Lit, check out the other guides in this repository. 
```
